app/scripts/app.js

```diff
@@ -3,6 +3,8 @@ import ReactDOM from 'react-dom'
 import Router from 'react-router'
 import routes from './routes'
 
+require('../styles/app.less')
+
 if (process.env.NODE_ENV !== 'production') {
   window.uiDebug = require('debug')
```

Comment:
That depends if we want to bundle the whole `debug` module, as the current build system will actually not bundle with this setup